src/huggingface_hub/_multi_commits.py

```diff
@@ -140,10 +142,10 @@ def plan_multi_commits(
 
     </Tip>
     """
-    addition_commits: List[List[CommitOperationAdd]] = []
+    addition_commits: List[List[Union[CommitOperationAdd, CommitOperationCopy]]] = []
     deletion_commits: List[List[CommitOperationDelete]] = []
 
-    additions: List[CommitOperationAdd] = []
+    additions: List[Union[CommitOperationAdd, CommitOperationCopy]] = []
```

Comment:
(nit) "additions" are considered to be `CommitOperationAdd ` **only** in other parts of the code, so this creates a small inconsistency.

In `create_commit` I kept "additions" for `CommitOperationAdd` and added "copies" for `CommitOperationCopy`

Reply:
Yep true :). Will make the change later once this PR is stable